flask_project/campaign_manager/test/test_models.py

```diff
@@ -2,134 +2,101 @@
 from unittest import TestCase
 from campaign_manager.models.models import *
 from sqlalchemy.orm import sessionmaker
+from campaign_manager.sqlalchemy_session import engine, session, Base
 
 
+# Creates a Test Database
 def db_setUp():
-    db_location = os.environ['TESTDATABASE_URL']
-    engine = create_engine(db_location, echo=True)
-    return engine
+    Base.metadata.drop_all(engine)
+    Base.metadata.create_all(engine)
 
 
 # Destroy the mock test Database
-def db_destroy(engine):
+def db_destroy(engine_test):
     for tbl in reversed(meta.sorted_tables):
         engine.execute(tbl.delete())
 
 
-# Create a session for the Database
-def create_session(engine):
-    Session = sessionmaker(bind=engine)
-    Session.configure(bind=engine)
-    session = Session()
-    return session
+db_setUp()
 
-
-engine_test = db_setUp()
-Base.metadata.create_all(engine_test)
-session = create_session(engine_test)
+user = User()
```

Comment:
Why do you initialise object?
The variables `user`, `campaign`, ... are not used later in the code.